src/Microsoft.Identity.Client/Cache/Items/MsalCacheItemBase.cs

```diff
@@ -57,5 +47,26 @@ internal void InitUserIdentifier()
                 HomeAccountId = ClientInfo.ToAccountIdentifier();
             }
         }
+
+        internal virtual void PopulateFieldsFromJObject(JObject j)
```

Comment:
Should this simply have been FromJObject?

Reply:
Typically, FromJObject() would be a static constructor returning a type.  This is a "send me a partial object and i'll fill it in" so I wanted to be clearer for readability.  happy for feedback here if folks disagree.